go/felix/ifacemonitor/iface_monitor.go

```diff
@@ -92,7 +107,70 @@ func (m *InterfaceMonitor) handleNetlinkUpdate(update netlink.LinkUpdate) {
 	m.storeUpdateAndNotifyOnChange(ifaceExists, attrs)
 }
 
+func (m *InterfaceMonitor) handleNetlinkAddrUpdate(update netlink.AddrUpdate) {
+	addr := update.LinkAddress.IP.String()
+	ifIndex := update.LinkIndex
+	exists := update.NewAddr
+	log.WithFields(log.Fields{
+		"addr":    addr,
+		"ifIndex": ifIndex,
+		"exists":  exists,
+	}).Info("Netlink address update.")
+
+	_, ifaceKnown := m.ifaceName[ifIndex]
+	if !ifaceKnown {
+		log.WithField("ifIndex", ifIndex).Warn("No known iface with this index.")
+		return
```

Comment:
Yep, I've made markups to cover this, by getting the complete list of addrs when we hear about a new up interface.